plugins/modules/hetzner_dns_record_set.py

```diff
@@ -129,10 +129,10 @@
     zone: foo.com
     record: foo.com
     type: CAA
-    ttl: 3600
+    ttl: null
```

Comment:
I would simply omit `ttl`. `null` is not a valid integer, and while that is not an error at the moment, it might become one in the future (as it currently is with `str` options).
```suggestion
```